src/librustc/session/config.rs

```diff
@@ -869,6 +881,8 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
         "disable the use of the redzone"),
     relocation_model: Option<String> = (None, parse_opt_string, [TRACKED],
          "choose the relocation model to use (rustc --print relocation-models for details)"),
+    relro_level: Option<RelroLevel> = (None, parse_relro_level, [TRACKED],
```

Comment:
I don't mind the new default now that I know `-Clink-arg` can still undo it all if needed.